samples/ipc_call_interface/inc/math_tlb.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
+
+#include "az_ulib_ipc_api.h"
+#include "ulib_result.h"
+#include <stdint.h>
+
+/*
+ * interface definition
+ */
+#define MATH_INTERFACE_NAME "math"
+#define MATH_INTERFACE_VERSION 1
+typedef az_ulib_ipc_interface_handle math_handle;
+
+enum math_interface_index {
```

Comment:
>enum math_interface_index [](start = 0, length = 25)

any reason why this isn't typedef'd

Reply:
In this case, it does not make difference, but I agree that we should do it as good practice. I'll fix it.

---
In reply to: [338710063](https://github.com/Azure/azure-ulib-c/pull/40#discussion_r338710063) [](ancestors = 338710063)